invenio_vocabularies/services/service.py

```diff
@@ -94,3 +94,21 @@ def search_request(self, identity, params, record_cls, **kwargs):
             params['_type'] = vocabulary_type
         return super().search_request(
             identity, params, record_cls, **kwargs)
+
+    def rebuild_index(self, identity):
```

Comment:
i think this method would be a good candidate for adding to the `invenio-records-resources` records service interface?

Reply:
and we *could* then introduce a new permission `can_reindex = [SystemProcess()]` that gets checked there... although that probably adds more complexity than benefits, as this functionality isn't intended to be exposed to the web.
what do you think?

Reply:
Fully agree. Will you move the method to records-resources?

Reply:
yes, i'll create a PR and link it to the same issue